### PR TITLE
Added task6.js

### PR DESCRIPTION
**Pull request by AnnetCC**

### CODE REVIEW

**Comment by sttaran** [comment]
1. new branch should be created for this HW
2. new directory should be created for this home work
3. each task should be written in different file

**Comment by AnnetCC** [comment]
> * new branch should be created for this HW
> * new directory should be created for this home work
> * each task should be written in different file

got it, thanks

File: js_basics/task6.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    for (let index = 1; index <= 10; index++) {
+        console.log(`${number} * ${index} = ${Math.imul(number, index)}`);
+    }
+
+// #3.2
+    console.log(colorPrint.underline("Exercise number 3.2"));
+
+    let index = 1;
+    while (index < 11) {
+        console.log(`${number} * ${index} = ${Math.imul(number, index)}`);
```
Comment (sttaran) [inline js_basics/task6.js:70]: It is nice to see you can learn by your own. I appreciate it. But to be honest `Math.imul` is kind of over engineering for this task. Using `*` is much shorter and more readable

Comment (sttaran) [reply js_basics/task6.js:70]: for both loops

**Review by sttaran — CHANGES_REQUESTED** [review]

**Pull request merged**